src/main/scala/rps/Game.scala

```diff
@@ -0,0 +1,51 @@
+package rps;
```

Comment:
super minor, but in scala it's convention not to use semicolons.

Reply:
You did well! Thank you for pointing that out!